### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by abergeron**
Some minor fixes that are accumulated in my local tree

### CODE REVIEW

File: theano/printing.py [hunk 1]
```diff
@@ -216,6 +216,8 @@ class Print(Op):
     """
     view_map = {0: [0]}
 
+    __props__ = ('message', 'attrs')
```
Comment (nouiz) [inline theano/printing.py]: I would put global_fn in that list.

**Comment by nouiz** [comment]
other then my comment. I'm good with this PR.

File: theano/printing.py (hunk 1 above)
Comment (abergeron) [inline theano/printing.py]: Maybe it was a bug, but the previous \__eq__ and \__hash__ didn't look at it.

Comment (abergeron) [inline theano/printing.py]: Actually, yes it was a bug.  I'll change it.

**Comment by nouiz** [comment]
when travis pass, you can merge.

**Pull request merged**